### DISCUSSION TITLE
Keyboards that work with PS2 programmed atmega8 microcontroller

### DISCUSSION THREAD

**Discussion by banman11** [Hartland/C64-Keyboard#5] [General]
Hello,

I thought I would start a separate thread off documenting keyboards that appear to operate with Heartland's unmodified version using the PS2 software library and an atmega8 microcontroller.

**Comment by banman11** [comment]
Feel free to make comments or add your own observations.

**Comment by Hartland** [comment]
The PS2 keyboard I was having issues with: Perixx Perboard 117,

I found that adding 220 ohm pull up resistors between Data/VCC and Clock/VCC solved the issue.

Further info: The keyboard has a SX830830 EKK8011 keyboard controller.
